src/common/imagpng.cpp

```diff
@@ -737,6 +775,34 @@ bool wxPNGHandler::SaveFile( wxImage *image, wxOutputStream& stream, bool verbos
         png_set_pHYs( png_ptr, info_ptr, resX, resY, PNG_RESOLUTION_METER );
 
     png_set_sBIT( png_ptr, info_ptr, &sig_bit );
+
+    // save "Description" text chunk
+#if defined(PNG_WRITE_iTXt_SUPPORTED) || defined(PNG_WRITE_tEXt_SUPPORTED)
+    if (image->HasOption(wxIMAGE_OPTION_PNG_DESCRIPTION))
+    {
+        const wxString& description = image->GetOption(wxIMAGE_OPTION_PNG_DESCRIPTION);
+
+        png_text text;
+        text.key = const_cast<char*>(wxIMAGE_OPTION_PNG_DESCRIPTION_KEY);
+        text.lang = nullptr;
+        text.lang_key = nullptr;
+
+#ifdef PNG_WRITE_iTXt_SUPPORTED
+        text.compression = PNG_ITXT_COMPRESSION_NONE;
+        text.text = const_cast<char*>(description.utf8_str().data());
+        text.text_length = 0;
+        text.itxt_length = strlen(text.text);
+#else
+        text.compression = PNG_TEXT_COMPRESSION_NONE;
+        text.text = const_cast<char*>(description.mb_str(wxConvISO8859_1).data());
```

Comment:
This is inconsistent with the use of `From8BitData()` above, either we should use `To8BitData()` here or use `wxConvISO8859_1` above as well.